> Could you please show failed test log? Maybe we need to report it upstream. I have no aarch64 machine to reproduce.

Actually I also don't have one, but maybe you can check it from ofborg (open https://github.com/NixOS/nixpkgs/pull/151799/checks?check_run_id=4610380444, then click "View more details on OfBorg" at the bottom of the page), so here it is:

```
=== RUN   Test_CalcScaleResolution
    tile_test.go:112: calcScaleResolution returned unexpected scale: 6.163099245746973e+08 expected: 6.163099245746973e+08
    tile_test.go:112: calcScaleResolution returned unexpected scale: 1.540774811436743e+08 expected: 1.540774811436743e+08
--- FAIL: Test_CalcScaleResolution (0.00s)
FAIL
FAIL    github.com/consbio/mbtileserver/handlers        0.052s
FAIL
error: builder for '/nix/store/i7fx7hkz4ar5q66hsjmvz8zcc9mziska-mbtileserver-0.8.0.drv' failed with exit code 1;
```

--- a/pkgs/servers/mbtileserver/default.nix
+++ b/pkgs/servers/mbtileserver/default.nix
@@ -1,17 +1,23 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
 
 buildGoModule rec {
   pname = "mbtileserver";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "consbio";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-QdirExVv7p7Mnhp8EGdTVRlmEiYpzoXVQbtO8aS9Kas=";
+    sha256 = "sha256-0wSc2DIfK6o3kSiH2sSQcYRR5dHnQbnJC6SX6DwVk1c=";
   };
 
-  vendorSha256 = "sha256-mUUxUZn8out6WNvKJKHoz+R44RDB0oWJb+57w72+E5w=";
+  # https://github.com/consbio/mbtileserver/issues/130
+  postPatch = lib.optionalString stdenv.isAarch64 ''
+    substituteInPlace handlers/tile_test.go \
+      --replace "Test_CalcScaleResolution" "Skip_CalcScaleResolution"
+  '';
+
+  vendorSha256 = "sha256-36tUTZud0hxH9oZlnKxeK/xzoEzfw3xFMnd/r0srw6U=";
 
   meta = with lib; {
     description = "A simple Go-based server for map tiles stored in mbtiles format";

